Cmdline/Action/KSP.cs

```diff
@@ -77,6 +96,13 @@ internal class AddOptions : CommonOptions
             [ValueOption(1)] public string path { get; set; }
         }
 
+        internal class CloneOptions : CommonOptions
+        {
+            [ValueOption(0)] public string nameOrPath { get; set; }
+            [ValueOption(1)] public string new_name { get; set; }
+            [ValueOption(1)] public string new_path { get; set; }
```

Comment:
One of these should probably be `ValueOption(2)`

Reply:
Interesting, what exactly does this number do? Because my tests worked despite the typo...

Reply:
It should be the index of that option in the list. Maybe it's got some sanity checking built in (hard to check the code for this dependency because it's been completely changed since the version CKAN uses).

Reply:
Okay, thanks